.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+	"presets": ["env"],
```

Comment:
from the docs:` babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together).`
Is this what we want?

Reply:
They have deprecated `babel-preset-es2015` and advised to use `babel-preset-env`

Reply:
ok but what's that do to the file size?